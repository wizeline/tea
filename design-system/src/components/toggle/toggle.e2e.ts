import { newE2EPage } from '@stencil/core/testing';

describe('wz-toggle', () => {
  it('should render a toggle-component', async () => {
    const page = await newE2EPage();
    await page.setContent('<wz-toggle />');
    const toggle = await page.find('wz-toggle');
    expect(toggle.outerHTML).toMatchSnapshot();
  });
});
