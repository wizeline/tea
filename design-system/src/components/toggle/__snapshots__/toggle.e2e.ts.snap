// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`wz-toggle should render a toggle-component 1`] = `"<wz-toggle class=\\"hydrated\\"><div class=\\"checkbox\\"><input id=\\"wz-toggle\\" type=\\"checkbox\\"><label for=\\"wz-toggle\\"></label></div></wz-toggle>"`;
